<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pg_twoplayers</class>
 <widget class="QWidget" name="pg_twoplayers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="cheatButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Cheat</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cooperateButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>440</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Cooperate</string>
   </property>
  </widget>
  <widget class="QLabel" name="left">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/creature.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="right_copycat">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>260</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/copycat_right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="coin_left">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>50</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/coin.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="coin_right">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>300</y>
     <width>50</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/coin.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="machine">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>130</y>
     <width>598</width>
     <height>417</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/machine_update.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="machine_00">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>130</y>
     <width>598</width>
     <height>417</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/machine_00.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="machine_10">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>130</y>
     <width>598</width>
     <height>417</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/machine_10.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="machine_01">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>130</y>
     <width>598</width>
     <height>417</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/machine_01.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="machine_11">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>130</y>
     <width>598</width>
     <height>417</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/machine_11.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="left_sad">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>255</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/sad.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="right_cheater">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>260</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/cheater_right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="left_happy">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>255</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/happy.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="right_cooperator">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>260</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/cooperator_right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="right_grudger">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>260</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/grudger_right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="right_detective">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>260</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/detective_right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="score_user">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="score_opponent">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="left_excited">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>255</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/excited.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="left_hate">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>255</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/hate.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>machine_11</zorder>
  <zorder>machine_10</zorder>
  <zorder>machine_00</zorder>
  <zorder>coin_left</zorder>
  <zorder>coin_right</zorder>
  <zorder>machine</zorder>
  <zorder>cooperateButton</zorder>
  <zorder>cheatButton</zorder>
  <zorder>machine_01</zorder>
  <zorder>right_copycat</zorder>
  <zorder>left</zorder>
  <zorder>left_sad</zorder>
  <zorder>right_cheater</zorder>
  <zorder>left_happy</zorder>
  <zorder>right_cooperator</zorder>
  <zorder>right_grudger</zorder>
  <zorder>right_detective</zorder>
  <zorder>score_user</zorder>
  <zorder>score_opponent</zorder>
  <zorder>left_excited</zorder>
  <zorder>left_hate</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
